.circleci/config.yml.j2

```diff
@@ -92,14 +95,15 @@ workflows:
             branches:
               only:
                 - master
-{% for distribution in distributions %}
+      {%- for distribution in distributions %}
       - build:
           name: build-{{ airflow_version }}-{{ distribution }}
           airflow_version: {{ airflow_version }}
           distribution_name: {{ distribution }}
           dev_build: {{ dev_build }}
           {%- if "dev" in ac_version and airflow_version not in dev_allowlist %}
-          extra_args: "--build-arg VERSION=$(curl https://pip.astronomer.io/simple/astronomer-certified/latest-{{ airflow_version | replace('.dev', '') }}.build)"
+          extra_args:
+            --build-arg VERSION=$(curl https://pip.astronomer.io/simple/astronomer-certified/latest-{{ airflow_version | replace('.dev', '') }}.build.json | jq -r '.output.astronomer_certified.package.version')
```

Comment:
Just a note: we will have to remember adding the JSON building logic for the dev builds too

Reply:
Oh, good catch!